### ISSUE TITLE
[BUG/Help] 这两天更新版本后，chat的微调好像用不了了

### ISSUE THREAD

**Issue by onekb** [THUDM/ChatGLM-6B#532]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

前几天使用chat微调还是可以用的，那时候output文件是完整的包，而不是增量微调包。
这两天更新后，使用的还是项目自带的train_chat.sh，模型用的是int4。
output文件确实小了，但是却无法运行了，具体形式为运行以下代码
```python
from transformers import AutoTokenizer, AutoModel
tokenizer = AutoTokenizer.from_pretrained("/content/ChatGLM-6B/ptuning/output/chattm/checkpoint-50", trust_remote_code=True)
model = AutoModel.from_pretrained("/content/ChatGLM-6B/ptuning/output/chattm/checkpoint-50", trust_remote_code=True).half().cuda()
model = model.eval()
response, history = model.chat(tokenizer, "你好", history=[])
print(response)
```
报以下内容后无反应，至少5分钟。期间显存一直在上升
You should probably TRAIN this model on a down-stream task to be able to use it for predictions and inference.
The dtype of attention mask (torch.int64) is not bool
最终报错
2023-04-11 13:51:41.577016: W tensorflow/compiler/tf2tensorrt/utils/py_utils.cc:38] TF-TRT Warning: Could not find TensorRT

### Steps To Reproduce

-

### Environment

```markdown
colab pro 默认环境
```

**Comment by onekb** [comment]
笔记本如下 https://colab.research.google.com/drive/1fAW40o-8Aor-5WGMCdeS0Fm-nqCc5-tg?usp=sharing
json内容截图
![image](https://user-images.githubusercontent.com/16450084/231198357-08b0ae20-a01f-4e2e-8c74-d55839919fb4.png)

**Comment by onekb** [comment]
修好了，参考了ptuning/main.py的evaluate处理。README可能要补充下怎么关联checkpoint
```python
import os
from transformers import AutoTokenizer, AutoModel, AutoConfig
import torch

model_name = "THUDM/chatglm-6b" # 模型名 或 模型路径
checkpoint_path = "checkpoint-1000" # 模型checkpoint路径
pre_seq_len = 8 # 模型前缀长度 跟你训练的PRE_SEQ_LEN一致

tokenizer = AutoTokenizer.from_pretrained(model_name, trust_remote_code=True)

config = AutoConfig.from_pretrained(model_name, trust_remote_code=True)
config.pre_seq_len = pre_seq_len
model = AutoModel.from_pretrained(model_name, config=config, trust_remote_code=True)

prefix_state_dict = torch.load(os.path.join(checkpoint_path, "pytorch_model.bin"))

new_prefix_state_dict = {}
for k, v in prefix_state_dict.items():
    new_prefix_state_dict[k[len("transformer.prefix_encoder."):]] = v
model.transformer.prefix_encoder.load_state_dict(new_prefix_state_dict)
model.half().cuda()

print(model.chat(tokenizer, "你好"))
```

**Comment by heccxixi** [comment]
您好，我想问一下Lora的权重文件该怎么更新呢
